docs/reference/api/docker_remote_api_v1.24.md

```diff
@@ -3926,33 +3926,68 @@ Create a service
     Content-Type: application/json
 
     {
-      "Name": "redis",
+      "Name": "web",
       "TaskTemplate": {
         "ContainerSpec": {
-          "Image": "redis"
+          "Image": "nginx:alpine",
+          "Mounts": [
+            {
+              "ReadOnly": true,
+              "Source": "web-data",
+              "Target": "/usr/share/nginx/html",
+              "Type": "VOLUME",
```

Comment:
This should be lowercase... 

https://github.com/docker/engine-api/blob/master/types/swarm/container.go#L25